srv/README.md

```diff
@@ -1,6 +1,19 @@
 # Demo server
 
-## Getting started
+## Getting started (Docker Compose)
+
+1. From inside the `/srv` directory, run Docker Compose as follows:
+
+```shell
+docker-compose up
+```
```

Comment:
Thanks for setting up `docker-compose` too. It is no doubt one step more convenient in this way.